subsys/net/lib/zperf/zperf_tcp_receiver.c

```diff
@@ -184,34 +178,35 @@ static void tcp_server_session(void)
 			/* Use Setting IP */
 			ret = zperf_get_ipv6_addr(MY_IP6ADDR,
 						  MY_PREFIX_LEN_STR,
-						  &in6_addr_my->sin6_addr);
+						  &in6_addr->sin6_addr);
 			if (ret < 0) {
 				NET_WARN("Unable to set IPv6");
 				goto use_existing_ipv6;
 			}
 		} else {
 		use_existing_ipv6:
 			/* Use existing IP */
-			in6_addr = zperf_get_default_if_in6_addr();
-			if (!in6_addr) {
-				NET_ERR("Unable to get IPv4 by default");
+			const struct in6_addr *addr =
+				zperf_get_default_if_in6_addr();
+			if (!addr) {
+				NET_ERR("Unable to get IPv6 by default\n");
```

Comment:
Why adding `\n` though? It'll print an extra empty line in the log.

Reply:
Sure I'll remove `\n` from all the logs

Reply:
done